AdminSelfUpgrade.php

```diff
@@ -3961,59 +4054,59 @@ private function _getJsErrorMsgs()
-txtError[38] = "'.$this->l('The uploaded file exceeds the upload_max_filesize directive in php.ini').'";
-txtError[39] = "'.$this->l('The uploaded file exceeds the MAX_FILE_SIZE directive that was specified in the HTML form').'";
-txtError[40] = "'.$this->l('The uploaded file was only partially uploaded').'";
-txtError[41] = "'.$this->l('No file was uploaded.').'";
-txtError[42] = "'.$this->l('Missing a temporary folder').'";
-txtError[43] = "'.$this->l('Failed to write file to disk').'";
-txtError[44] = "'.$this->l('File upload stopped by extension').'";
-txtError[45] = "'.$this->l('Cannot convert your database\'s data to utf-8.').'";
-txtError[46] = "'.$this->l('Invalid shop name').'";
-txtError[47] = "'.$this->l('Your firstname contains some invalid characters').'";
-txtError[48] = "'.$this->l('Your lastname contains some invalid characters').'";
-txtError[49] = "'.$this->l('Your database server does not support the utf-8 charset.').'";
-txtError[50] = "'.$this->l('Your MySQL server does not support this engine, please use another one like MyISAM').'";
-txtError[51] = "'.$this->l('The file /img/logo.jpg is not writable, please CHMOD 755 this file or CHMOD 777').'";
-txtError[52] = "'.$this->l('Invalid catalog mode').'";
-txtError[999] = "'.$this->l('No error code available').'";
+txtError[0] = "'.$this->trans('Required field', array(), 'Modules.Autoupgrade.Admin').'";
+txtError[1] = "'.$this->trans('Too long!', array(), 'Modules.Autoupgrade.Admin').'";
+txtError[2] = "'.$this->trans('Fields are different!', array(), 'Modules.Autoupgrade.Admin').'";
+txtError[3] = "'.$this->trans('This email address is wrong!', array(), 'Modules.Autoupgrade.Admin').'";
+txtError[4] = "'.$this->trans('Impossible to send the email!', array(), 'Modules.Autoupgrade.Admin').'";
+txtError[5] = "'.$this->trans('Cannot create settings file, if /app/config/parameters.php exists, please give the public write permissions to this file, else please create a file named parameters.php in config directory.', array(), 'Modules.Autoupgrade.Admin').'";
+txtError[6] = "'.addslashes($this->trans('Cannot write settings file, please create a file named settings.inc.php in the "config" directory.', array(), 'Modules.Autoupgrade.Admin')).'";
+txtError[7] = "'.$this->trans('Impossible to upload the file!', array(), 'Modules.Autoupgrade.Admin').'";
+txtError[8] = "'.$this->trans('Data integrity is not valided. Hack attempt?', array(), 'Modules.Autoupgrade.Admin').'";
+txtError[9] = "'.$this->trans('Impossible to read the content of a MySQL content file.', array(), 'Modules.Autoupgrade.Admin').'";
+txtError[10] = "'.$this->trans('Cannot access a MySQL content file.', array(), 'Modules.Autoupgrade.Admin').'";
+txtError[11] = "'.$this->trans('Error while inserting data in the database:', array(), 'Modules.Autoupgrade.Admin').'";
+txtError[12] = "'.$this->trans('The password is incorrect (alphanumeric string at least 8 characters).', array(), 'Modules.Autoupgrade.Admin').'";
```

Comment:
The password is incorrect (must be alphanumeric string with at least 8 characters) // Install